include/mata/parser/re2parser.hh

```diff
@@ -8,14 +8,23 @@
 
 #include "mata/nfa/nfa.hh"
 
+
+// Encoding for the regular expression
+// FIXME: Use enum class re2::Regexp::ParseFlags from re2/regexp.h instead. It is not possible to include it here. Need to fix cmake.
+enum Encoding {
+    UTF8 = 0,
+    Latin1 = 1<<5
+};
```

Comment:
An idea would also be to use `enum class` here, and only in the src code parse the passed encoding as the RE2 `Encoding` enum. It would even be cleaner so that the user does not have to concern themselves with the underlying values and simply use `Encoding::Latin1` with type checking. Otherwise, passing random integers will not generate an error.

Reply:
Good idea. I'll change it to `enum class`.